### PR TITLE
add Capital One and ING case studies

### PR DESCRIPTION
**Pull request by alexcontini**
@kbarnard10 and @KimMcMahon I have added ING and Capital One case studies for your review!

### CODE REVIEW

File: content/en/case-studies/ing.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    Company &nbsp;<b>ING</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Location &nbsp;<b>Amsterdam, Netherlands
+</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Industry &nbsp;<b>Finance</b>
+</div>
+
+<hr>
+<section class="section1">
+<div class="cols">
+  <div class="col1" style="width:95% !important;">
+    <h2>Challenge</h2>
+    After undergoing an agile transformation, <a href="https://www.ing.com/">ING</a> realized it needed a standardized platform to support the work their developers were doing. "Our DevOps teams got empowered to be autonomous," says Infrastructure Architect Thijs Ebbers. "It has benefits; you get all kinds of ideas. But a lot of teams are going to devise the same wheel. Teams started tinkering with <a href="https://www.docker.com/">Docker</a>, Docker Swarm, <a href="https://kubernetes.io/">Kubernetes</a>, < a href="https://mesosphere.com/">Mesos</a>. Well, it’s not really useful for a company to have one hundred wheels, instead of one good wheel.
```
Comment (kbarnard10) [inline content/en/case-studies/ing.html]: Mesosphere link is broken, looks like there is an extra space

File: content/en/case-studies/_index.html [hunk 2]
```diff
@@ -63,6 +61,9 @@ <h4><i>"Kubernetes has the opportunity to be the new cloud platform. The amount
     <main>
         <h3>Kubernetes Users</h3>
         <div id="usersGrid">
+         
+          <a target="_blank" href="/case-studies/ing/"><img src="/images/case_studies/ing_feature.png" alt="ING"></a>
+          <a target="_blank" href="/case-studies/capital-one/"><img src="/images/case_studies/capitalone_feature.png" alt="Capital One"></a>
           <a target="_blank" href="/case-studies/northwestern-mutual/"><img src="/images/case_studies/northwestern_feature.png" alt="Northwestern Mutual"></a>
```
Comment (kbarnard10) [inline content/en/case-studies/_index.html]: Could you double check that all the of logos in the bottom section are the same size. I think the latter 3 in the first row are larger.

**Comment by kbarnard10** [comment]
@alexcontini Add 2 comments. Could you also update the case studies on the homepage to reflect the most recent 4?

**Comment by alexcontini** [comment]
@kbarnard10 all updates have been made, I think @zacharysarah is needed to merge because of the images?

**Pull request merged**